### DISCUSSION TITLE
Failed to initialize worker from different origin using Webpack plugin

### DISCUSSION THREAD

**Discussion by c3Vu** [perspective-dev/perspective#2160] [Q&A]
## Bug Report
![image](https://user-images.githubusercontent.com/71494005/228487260-b50a5d6e-be3e-4007-b78a-0e77789e21f3.png)

I am using Webpack and @finos/perspective-webpack-plugin to load the wasm/worker/js files. But it is not working when the worker script is from a different origin.

### Environment:
1.9.4

**Comment by texodus** [comment]
@c3Vu This is not a Perspective error - the browser does not allow "this".  Of course, I don't know what "this" is, because you ignored the [template instructions which asked you to provide a repro of your issue](https://github.com/finos/perspective/blob/master/.github/ISSUE_TEMPLATE/Bug_report.md).  I cannot debug a screenshot.
